data/distro.yml

```diff
@@ -720,4 +720,21 @@
         <p>Check that in the Flatpak section the box is checked.</p>
         <p>Note: with this Flathub app search will be integrated in Discover, if you want to limit the app search to Flathub you can mark Flatpak as default by clicking on the star.</p>
     </ol>
-    
+
+ - name: "GNU Guix"
+  logo: "guix.svg"
+  info: >
+    <ol class="distrotut">
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>Flatpak can be installed using GNU Guix's default repositories:</p>
+        <pre><code>
+          <span class="unselectable">#</span> guix install flatpak
+        </code></pre>
+      </li>
+      <li>
```

Comment:
What about Flathub? Is it added out-of-box in Guix or why is this step missing?

Reply:
Any chance someone could answer the question from previous comment? :-)

Reply:
I have tested Guix (by the way, it's Guix, not GUIX) and it looks like that it has no support for graphical management of software at all.